webapp/src/utils/lyraConfig.ts

```diff
@@ -18,6 +18,7 @@ const lyraConfigSchema = z.object({
   baseBranch: z.optional(z.string()),
   projects: z.array(
     z.object({
+      languages: z.optional(z.array(z.string()).min(1)),
```

Comment:
An alternative would be to add `languages` under `translations`, next to `path`. Is there any reason you prefer it here in the root?

Reply:
that because we have a lint rule to order alphabetically :/

Reply:
I was considering moving it inside transaltions, but after discussing it, we decided it's fine as it is.